### ISSUE TITLE
When placing unlit campfires (lit-when-placed: false) they shows lit for a second

### ISSUE THREAD

**Issue by lordrius** [pl3xgaming/Purpur#1387]
### Spark link

N/A

### Expected behavior

Not see the lit campfire and black flames when placed.

### Observed/Actual behavior

When placing unlit campfires (lit-when-placed: false) they shows lit for a second.

### Steps/models to reproduce

Set 'lit-when-placed: false' and place a campfire.

### Purpur version

[19:19:19] [Thread-10/INFO]: Current: git-Purpur-2014 (MC: 1.20.1)*
Previous: git-Purpur-2005 (MC: 1.20.1)

### Agreements

- [X] I am running the latest version of Purpur available from https://purpurmc.org/downloads.
- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [ ] I ticked all the boxes without actually reading them
- [X] My version of Minecraft is supported by Purpur.

### Other

I know that will be hard/impossible to fix it but would be nice to avoid this because it could be very annoying for some people. Try to place a lot of unlit campfires at night, those flashes kill you.

Thank you very much anyway!

**Comment by BillyGalbreath** [comment]
This is caused by client "prediction" and network latency. Your client thinks the campfire should be lit. After the client places a lit campfire it tells the server it placed it. The server then responds back to the client telling it to make it unlit, so the client puts the fire out. It's as simple as that.

TLDR; get better ping.

**Comment by lordrius** [comment]
I'm trying it at a localhost, my computer. Thanks for the explanation :)

**Comment by granny** [comment]
Billy explained it well - *technically* speaking it works as intended.
